aep/general/0122/aep.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: 122
+state: approved
+created: 2023-10-11
+placement:
+  category: resource-design
```

Comment:
are categories exposed publicly? If not, maybe we can use "specification" here?

Reply:
I thought they were...I think the meta vs. spec vs. guidance thing is a different thing?  But will leave this open in case others have opinions.

Reply:
ack, BTW PTAL at #65